libs/fixtures/src/data/electionMinimalExhaustiveSampleWithReportingUrl/electionMinimalExhaustiveSample.json.ts

```diff
@@ -0,0 +1,61 @@
+/* Generated by res-to-ts. DO NOT EDIT */
```

Comment:
Would it be possible to just change this fixture to have results reporting turned on and use a different fixture for results reporting turned off? Or did you need to add a new election fixture just to distinguish these two cases?

Reply:
I didn't want to do that and risk changing the hash of that election, with all the potential downstream effects.

Reply:
Gotcha, that makes sense, though I would hope our tests aren't dependent on the exact hash of an election definition. That would make them particularly brittle.

The reason I ask is because it seems like it could get pretty unmanageable to add a different fixture for every time we want to test one field change of an election definition.

Reply:
ok, I will explore if I can make this change safely

Reply:
sadly, it does cause other tests to fail, because we include the election hash in a number of directory structures. I don't know that that makes our tests brittle so much as we have inherent constraints given election hashes.

On the whole, my sense is we're better off adding that new election with reporting URL, since it is not longer the default, but I welcome pushback.

Reply:
My only opinion is that we should be able to make changes to test fixtures without breaking tests (unless we actually make a logical change that breaks the behavior that the test is testing). I think that should be possible even with strict election hash checking.

Maybe you don't want to get blocked on that in this PR, which is reasonable, and I'm happy to approve.

Reply:
@jonahkagan I don't think we have that capability right now, given how tightly everything is bound to the election definition. It would be a significant effort to change this. I'll file a ticket for us to consider:

https://github.com/votingworks/vxsuite/issues/2247

Reply:
Sounds good! Thanks